2021-02-16 21:41:54,157 INFO    MainThread:140208 [wandb_setup.py:_flush():70] setting env: {}
2021-02-16 21:41:54,158 INFO    MainThread:140208 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-02-16 21:41:54,158 INFO    MainThread:140208 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-16 21:41:54,158 INFO    MainThread:140208 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-16 21:41:54,158 INFO    MainThread:140208 [wandb_init.py:_log_setup():303] Logging user logs to /home/tyna/Documents/safe-experts/algos/wandb/run-20210216_214154-zjg27cqz/logs/debug.log
2021-02-16 21:41:54,158 INFO    MainThread:140208 [wandb_init.py:_log_setup():304] Logging internal logs to /home/tyna/Documents/safe-experts/algos/wandb/run-20210216_214154-zjg27cqz/logs/debug-internal.log
2021-02-16 21:41:55,742 INFO    MainThread:140208 [wandb_run.py:_console_start():1237] atexit reg
2021-02-16 21:41:55,743 INFO    MainThread:140208 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.REDIRECT
2021-02-16 21:41:55,743 INFO    MainThread:140208 [wandb_run.py:_redirect():1107] Redirecting console.
2021-02-16 21:41:55,743 INFO    MainThread:140208 [redirect.py:install():204] install start
2021-02-16 21:41:55,747 INFO    MainThread:140208 [redirect.py:install():219] install stop
2021-02-16 21:41:55,747 INFO    MainThread:140208 [redirect.py:install():204] install start
2021-02-16 21:41:55,754 INFO    MainThread:140208 [redirect.py:install():219] install stop
2021-02-16 21:41:55,754 INFO    MainThread:140208 [wandb_run.py:_redirect():1151] Redirects installed.
2021-02-16 21:41:56,406 INFO    MainThread:140208 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 1
2021-02-16 21:41:56,406 INFO    MainThread:140208 [wandb_run.py:_restore():1179] restore
2021-02-16 21:41:56,406 INFO    MainThread:140208 [redirect.py:uninstall():223] uninstall start
2021-02-16 21:41:56,406 INFO    MainThread:140208 [redirect.py:_stop():278] _stop: stdout
2021-02-16 21:41:56,406 INFO    stdout    :140208 [redirect.py:_pipe_relay():120] relay done saw last write: stdout
2021-02-16 21:41:56,406 INFO    stdout    :140208 [redirect.py:_pipe_relay():136] relay done done: stdout
2021-02-16 21:41:56,407 INFO    MainThread:140208 [redirect.py:_stop():284] _stop closed: stdout
2021-02-16 21:41:56,407 INFO    MainThread:140208 [redirect.py:_stop():290] _stop joined: stdout
2021-02-16 21:41:56,407 INFO    MainThread:140208 [redirect.py:_stop():292] _stop rd closed: stdout
2021-02-16 21:41:56,407 INFO    MainThread:140208 [redirect.py:uninstall():227] uninstall done
2021-02-16 21:41:56,407 INFO    MainThread:140208 [redirect.py:uninstall():223] uninstall start
2021-02-16 21:41:56,407 INFO    MainThread:140208 [redirect.py:_stop():278] _stop: stderr
2021-02-16 21:41:56,407 INFO    MainThread:140208 [redirect.py:_stop():284] _stop closed: stderr
2021-02-16 21:41:56,407 INFO    stderr    :140208 [redirect.py:_pipe_relay():120] relay done saw last write: stderr
2021-02-16 21:41:56,407 INFO    stderr    :140208 [redirect.py:_pipe_relay():136] relay done done: stderr
2021-02-16 21:41:56,407 INFO    MainThread:140208 [redirect.py:_stop():290] _stop joined: stderr
2021-02-16 21:41:56,407 INFO    MainThread:140208 [redirect.py:_stop():292] _stop rd closed: stderr
2021-02-16 21:41:56,408 INFO    MainThread:140208 [redirect.py:uninstall():227] uninstall done
2021-02-16 21:41:56,426 INFO    MainThread:140208 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "1e4c13329f1d4e44b0821af4e7f851a3"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 747
      total_bytes: 747
    }
  }
}

2021-02-16 21:41:58,433 INFO    MainThread:140208 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "0b1bc68e35e447c3938761b97ba2a08c"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 8429
      total_bytes: 8429
    }
  }
}

2021-02-16 21:41:59,561 INFO    MainThread:140208 [wandb_run.py:_show_files():1536] logging synced files
2021-02-16 21:41:59,563 INFO    MainThread:140208 [internal.py:handle_exit():138] Internal process exited
